apps/predbat/predbat.py

```diff
@@ -43,6 +43,56 @@
 
 INVERTER_TYPES = {"GE": "GivEnergy", "GS": "Ginlong Solis", "SE": "SolarEdge", "SX4": "Solax Gen4 (Modbus Power Control)"}
 
+# Each group uses the defaults set in CONFIG_ITEMS unless specified
+# EXCEPT 'Custom' which does not change anything
+CONFIG_GROUPS = {
+    "Custom": {},
+    "Eco7 | No Export": {},
+    "Eco 7 | Octopus Export": {
+        "calculate_best_discharge": True,
+        "calculate_discharge_first": True,
+        "combine_charge_slots": True,
+        "combine_discharge_slots": False,
+        "set_discharge_window": True,
+        "metric_min_improvement": 0,
+        "metric_min_improvement_discharge": 0,
+        "rate_high_threshold": 0,
+        "set_discharge_freeze": True,
+        "set_charge_freeze": False,
+        "calculate_max_windows": 96,
+        "predbat_metric_battery_cycle": 3,
+    },
+    "Octopus Flux / Cozy": {
+        "calculate_best_discharge": True,
+        "calculate_discharge_first": True,
+        "combine_charge_slots": True,
+        "combine_discharge_slots": True,
+        "set_discharge_window": True,
+        "metric_min_improvement": 0,
+        "metric_min_improvement_discharge": 0.1,
+        "rate_low_threshold": 0.8,
+        "rate_high_threshold": 1.2,
+        "metric_battery_cycle": 2,
+        "set_discharge_freeze": True,
+        "set_charge_freeze": True,
+    },
+    "Octopus Agile": {
+        "calculate_best_discharge": True,
+        "calculate_discharge_first": True,
+        "combine_discharge_slots": False,
+        "set_discharge_window": True,
+        "metric_min_improvement": 0,
+        "metric_min_improvement_discharge": 0.1,
+        "rate_low_threshold": 0,
+        "rate_high_threshold": 0,
+        "metric_battery_cycle": 2,
+        "calculate_max_windows": 32,
```

Comment:
Probably should be max_windows 40 now and metric_battery_cycle 0 (users are confused otherwise)

Reply:
Fixed